public/javascript/main.js

```diff
@@ -377,4 +377,16 @@ $(function(){
           $(this).data('chart', chart);
       }
   })
+
+  // Event tracking
+  // Clicks on elements with data-ga-track-click attribute will be recorded
+  function trackClick(eventDescription){
+    eventDescription = eventDescription || 'Undescribed event.';
+    ga('send', 'event', 'user interaction', eventDescription, document.title);
```

Comment:
The JS isn't going to work at the moment as some of the interactions navigate away from the page. [Knowing when the hit has been sent](https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#transport)